source/saveasdialog.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <QDialog>
+#include <QJsonObject>
+
+class D1CelBase;
+
+enum class SAVE_CLIPPING_TYPE {
+    CLIPPED_AUTODETECT,
```

Comment:
Should this be `CLIPPED_PRESERVED`?

Reply:
I do not know yet. Maybe it can be merged with `OPEN_CLIPPING_TYPE` later. Maybe one of them needs to be extended later... I think it should be decided after some feedback from the users.